### ISSUE TITLE
request the minimize command?

### ISSUE THREAD

**Issue by yuomtheara** [MohammadYounes/AlertifyJS#52]
request the minimize command?

**Comment by MohammadYounes** [comment]
Minimize to where ?

**Comment by yuomtheara** [comment]
Minimize to bottom of the left with mini title bar.

**Comment by alessandrofrancesconi** [comment]
I personally think that such feature is a bit "out of scope". Alertify should be intended as a tool to show _alerts_ and, by definition, they should call the user's attention to something for a very short lifetime.
If you introduce minimizing, such effect can be lost.

**Comment by MohammadYounes** [comment]
I agree, but this repo outgrown the concept of alerts, with dialog factory it's possible to create any kind of dialog and features such as minimize or even docking would still fit.

One way to come around this might be by splitting the lib into a lite version (for basic stuff) and full version with the more sophisticated feature set .

**Comment by stoivo** [comment]
I like you idea @MohammadYounes since we use Alertify js to show some report you our customers. I think it is correct to add more things to this, so it looks more like a dialog box then alert.
